higher_health/templates/healthcheck_questionnaire.html

```diff
@@ -112,8 +112,15 @@ <h3 class="heading heading-secondary">Registration</h3>
               <div class="collapsibles__label">
                 <h2 class="heading-secondary">Where are you going today?</h2>
               </div>
+              {% if form.destination_fields_has_errors %}
```

Comment:
I've added an error message title to each block since the one at the top and the bottom doesn't quite make sense and might be hidden in cases where there are error messages in other blocks.

Reply:
![image](https://user-images.githubusercontent.com/623417/85503399-e0198e00-b5e9-11ea-8b27-752db38216b3.png)